core/src/main/java/org/infinispan/transaction/impl/TotalOrderRemoteTransactionState.java

```diff
@@ -16,7 +16,9 @@
  *
  * @author Pedro Ruivo
  * @since 5.3
+ * @deprecated since 10.0. Total Order will be removed.
```

Comment:
If you deprecate this, you should probably deprecate `RemoteTransaction.getTransactionState()` as well.

Reply:
done!